packages/rps/src/redux/game-v2/saga.ts

```diff
@@ -34,74 +34,74 @@ const isPlayersTurnNext = (
 };
 
 export function* gameSaga(client: RPSChannelClient) {
-  while (true) {
-    yield take('*'); // run after every action
+  yield takeEvery('*', gameSagaRun, client);
```

Comment:
I believe the main difference is that now you can have several `gameSagaRun` tasks running concurrently? Is that desired?

(e.g. we can run another task without having to wait for the  previous wallet api call to resolve)

Reply:
Redux sagas don't run concurrently anyhow. I think it does mean that if the `gameSaga` emits events, it will run again on those events after it's done, which doesn't hurt. Mainly it seems nicer not to have a `while (true)` loop.